sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/OperationType.java

```diff
@@ -40,7 +40,7 @@
     QueryPlan,
     Stop,
     Throttle,
-    Update,
```

Comment:
where did update go? don't se support Update?

Reply:
Operation type update was renamed to Patch in Backend, .net. 
PR: https://msdata.visualstudio.com/DefaultCollection/CosmosDB/_git/CosmosDB/pullrequest/190858